cmd/earthly/satellite_cmds.go

```diff
@@ -404,6 +404,13 @@ func (app *earthlyApp) getSatelliteOrg(ctx context.Context, cloudClient *cloud.C
 		}
 		return app.orgName, orgID, nil
 	}
+	if orgName = app.cfg.Global.Org; orgName != "" {
```

Comment:
There's an `if app.OrgName != ""` block above, which means setting the `--org` flag or using the `EARTHLY_ORG` env var still takes precedence over the value set in the config file. If neither are set, we still run the old "guess" logic.

Reply:
nit: can you move `orgName = app.cfg.Global.` out of the `if`, to make it more clear it's assigning to the existing `orgName`.

(yes it will immediately be replaced once `orgName, orgID, err = cloudClient.GuessOrgMembership(ctx)` is hit, but the code immediately following the if might hypothetically change in the future.)